e2e-tests/tests/reserve/conftest.py

```diff
@@ -26,7 +26,7 @@ def governance_address(config: ApiConfig) -> str:
 
 
 @fixture(scope="session")
-def payment_key(config: ApiConfig, governance_skey_with_cli):
+def payment_key(config: ApiConfig, cardano_payment_key, governance_skey_with_cli):
```

Comment:
this is to ensure proper order in fixture setup, copies payment key to cardano-node container.